src/main/java/com/ita/if103java/ims/dto/WarehouseLoadDto.java

```diff
@@ -0,0 +1,52 @@
+package com.ita.if103java.ims.dto;
+
+import org.springframework.stereotype.Component;
+
+import javax.validation.constraints.NotNull;
+
+@Component
+public class WarehouseLoadDto {
+    @NotNull
```

Comment:
there is no sense in these validations here, because this Dto is not input form from the UI. It is use only to send data from server to client